src/components/calcite-loader/calcite-loader.tsx

```diff
@@ -38,20 +40,23 @@ export class CalciteLoader {
   //
   //--------------------------------------------------------------------------
   render() {
-    const id = this.el.id || this.guid;
-    const size = this.inline ? 20 : 56;
-    const radius = this.inline ? 9 : 25;
+    const { el, inline, scale, text, type, value } = this;
+
+    const id = el.id || guid;
```

Comment:
Do we need to use this at all here?

Reply:
I didn't see it being used. I can remove it.